dark/index.html

```diff
@@ -244,47 +295,5 @@ <h3>CONTACT <span class="text-info">ME</span></h3>
 
   <!-- Internal JavaScript -->
   
-  <script>

```

Comment:
Don't remove this script content, undo that change @Neutx

Reply:
Added it back, it was removed by mistake. Thanks for letting me know.